webpack/assets/javascripts/react_app/redux/API/APIRequest.js

```diff
@@ -46,6 +50,13 @@ export const apiRequest = async (
       response: response.data,
     });
 
+    updatedData &&
```

Comment:
I like the fact that this UPDATE action is triggered from within the API middleware.  If the action were more easily available, there would be a bigger risk of the Redux store and backend becoming out of sync.  But here, it provides an opinionated workflow and should avoid a lot of potential bugs. :+1: